packages/sak-app/src/behandling/BehandlingIndex.jsx

```diff
@@ -322,13 +323,14 @@ export const getFagsakInfo = createSelector(
 const mapStateToProps = state => {
   const behandlingId = getUrlBehandlingId(state);
   const behandlingType = getBehandlingerTypesMappedById(state)[behandlingId];
+  const kodeverk = erKlage(behandlingType) ? getAlleKlagekodeverk(state) : getAlleFpSakKodeverk(state);
   return {
     behandlingId,
     behandlingType,
     behandlingVersjon: getTempBehandlingVersjon(state),
     location: state.router.location,
     featureToggles: getFeatureToggles(state),
-    kodeverk: erTilbakekreving(behandlingType) ? getAlleFpTilbakeKodeverk(state) : getAlleFpSakKodeverk(state),
+    kodeverk: erTilbakekreving(behandlingType) ? getAlleFpTilbakeKodeverk(state) : kodeverk,
```

Comment:
Logikken for å bestemme riktig kodeverk blir litt unødvendig vanskelig å forstå her. På linje 326 finner man ut om det er klage, og hvis ikke brukes kodeverket fra `getAlleFpSakKodeverk`, men verdien fra `getAlleFpSakKodeverk` brukes bare gitt at det ikke er tilbakekreving (linje 333). Kunne vi samla den logikken på ett sted, og holdt den utenfor return-statementen slik at vi bare returnerer kodeverk, ville det blitt enklere å forstå 👍 

F.eks.
```
if (erKlage)
   kodeverk = kodeverketTilKlage
else if (erTilbakekreving)
   kodeverk = kodeverketTilTilbakekreving
else
   kodeverk = alleKodeverk
```

Reply:
Ja, det var dette ``getAlleKodeverkForBehandlingstype`` ([commit nr. 2](https://github.com/navikt/k9-sak-web/pull/303/commits/b220e2ad21bba0d8677db67dadea9fc46d8c8b99)) skulle løse. Har bare oversett denne kodesnutten - skal oppdatere her også.